src/main/scala/com/raphtory/core/config/telemetry/PartitionTelemetry.scala

```diff
@@ -0,0 +1,130 @@
+package com.raphtory.core.config.telemetry
```

Comment:
Think we need some data on the synced updates, a count of them at min -- how long each update takes to sync perhaps if its not too much of an overhead

Reply:
Added a counter for synced updates: `totalSyncedStreamWriterUpdates`